app/views/followups/_followup.html.erb

```diff
@@ -30,11 +30,9 @@
           <% if id.nil? && incoming_message.valid_to_reply_to? %>
             <li><%= link_to(_("the main FOI contact address for {{public_body}}", :public_body => name), new_request_followup_path(:request_id => @info_request.id)) %></li>
           <% else %>
-            <% if !id.nil? %>
+            <% if id.present? %>
               <% if @info_request.public_body.request_email == email %>
-                <% if !incoming_message.nil? %>
```

Comment:
we now always have an`incoming_message` at this point so we no longer need to check for it